src/main/js/controls/Bubble/BubbleMultipleDataset.js

```diff
@@ -0,0 +1,157 @@
+"use strict";
+import constants from "../../helpers/constants";
+import { prepareLabelShapeItem } from "../../helpers/label";
+import { reflowLegend } from "../../helpers/legend";
+import { createRegion } from "../../helpers/region";
+import styles from "../../helpers/styles";
+import utils from "../../helpers/utils";
+import {
+    clickHandler,
+    hoverHandler,
+    prepareLegendItems,
+    processDataPoints,
+    getDataPointValues,
+    calculateValuesRange
+} from "./helpers/helpers";
+import { draw, drawBubbles } from "./helpers/helpersMultipleDataset";
+import Bubble from "./Bubble";
+
+/**
+ * A Bubble graph is a graph used to represent a collection of data
+ * points connected by a Bubble along the X and Y Axis.
+ *
+ * Lifecycle functions include:
+ *  * Load
+ *  * Generate
+ *  * Unload
+ *  * Destroy
+ *
+ * @module Bubble
+ * @class BubbleMultipleDataset
+ */
+class BubbleMultipleDataset extends Bubble {
+    /**
+     * @inheritdoc
+     * @public
+     */
+    load(graph) {
+        if (graph.content.length > 7) {
+            console.warn(
+                "BubbleSingleDataset is limited to a maximum of 7 datasets"
+            );
```

Comment:
Will execution stops if we give more than 7 datasets?

Reply:
No, it'll just ignore any datasets after 7 and won't load them.